text/0000-line-width.md

```diff
@@ -0,0 +1,128 @@
+- Start Date: 2016-05-04
+- RFC PR: (leave this empty)
+- Implementation Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Define the maximum width of a line and the width of indentation.
+
+
+# Details
+[details]: #details
+
+The maximum width of a line is 100 characters. The maximum width of a comment
+(including any commenting characters, e.g., `//`, but not including any
+indentation) is 80 characters. A comment should be wrapped at the minimum of the
+80 character comment limit and the 100 character line limit.
+
+Indentation should use spaces, not tabs. Each level of indentation should be
+four characters wide.
```

Comment:
I know there's a sizable minority that wishes we would switch to two spaces. I'm not sure how the wider community feels about that.